<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>775</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  rgba(255, 255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToolPen"/>
   <addaction name="actionToolHighlight"/>
   <addaction name="actionToolRectangle"/>
   <addaction name="actionToolText"/>
   <addaction name="actionToolErease"/>
   <addaction name="separator"/>
   <addaction name="actionPenColorBlack"/>
   <addaction name="actionPenColorRed"/>
   <addaction name="actionPenColorBlue"/>
   <addaction name="actionPenColorGreen"/>
   <addaction name="actionPenColorYellow"/>
   <addaction name="separator"/>
   <addaction name="actionClearDrawings"/>
   <addaction name="separator"/>
   <addaction name="actionScreenshot"/>
   <addaction name="actionScreenShotClear"/>
   <addaction name="separator"/>
   <addaction name="actionOpenPDF"/>
   <addaction name="actionPdfNextPage"/>
   <addaction name="actionPdfPrevPage"/>
   <addaction name="separator"/>
   <addaction name="actionSaveImage"/>
   <addaction name="actionWindowsMenu"/>
  </widget>
  <widget class="QDockWidget" name="dockTimer">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>524287</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgba(100, 100, 100, 100);</string>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Timer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockTimerContents">
    <widget class="QPushButton" name="btnTimerStart">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTimer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTimerPause">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTimerReset">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>92</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeTimerAlert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm:ss</string>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>15</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindows">
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 0);s</string>
    </property>
    <property name="title">
     <string>Windows</string>
    </property>
   </widget>
   <addaction name="menuWindows"/>
  </widget>
  <action name="actionToolPen">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/pen</normaloff>:/icons/pen</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/screenshot</normaloff>:/icons/screenshot</iconset>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionScreenShotClear">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_screenshot_clear.png</normaloff>:/icons/images/icon_screenshot_clear.png</iconset>
   </property>
   <property name="text">
    <string>ScreenShot Clear</string>
   </property>
  </action>
  <action name="actionClearDrawings">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_clear.png</normaloff>:/icons/images/icon_clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Drawings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionPenColorBlack">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_black.png</normaloff>:/icons/images/icon_black.png</iconset>
   </property>
   <property name="text">
    <string>Pen Color Black</string>
   </property>
  </action>
  <action name="actionPenColorBlue">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_blue.png</normaloff>:/icons/images/icon_blue.png</iconset>
   </property>
   <property name="text">
    <string>Pen Color Blue</string>
   </property>
  </action>
  <action name="actionPenColorGreen">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_green.png</normaloff>:/icons/images/icon_green.png</iconset>
   </property>
   <property name="text">
    <string>Pen Color Green</string>
   </property>
  </action>
  <action name="actionPenColorRed">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_red.png</normaloff>:/icons/images/icon_red.png</iconset>
   </property>
   <property name="text">
    <string>Pen Color Red</string>
   </property>
  </action>
  <action name="actionPenColorYellow">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_yellow.png</normaloff>:/icons/images/icon_yellow.png</iconset>
   </property>
   <property name="text">
    <string>Pen Color Yellow</string>
   </property>
  </action>
  <action name="actionToolRectangle">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_draw_rectangle.png</normaloff>:/icons/images/icon_draw_rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionToolHighlight">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_highlighter.png</normaloff>:/icons/images/icon_highlighter.png</iconset>
   </property>
   <property name="text">
    <string>Highlight</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionToolText">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_draw_text.png</normaloff>:/icons/images/icon_draw_text.png</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_save.png</normaloff>:/icons/images/icon_save.png</iconset>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionToolErease">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_ereaser.png</normaloff>:/icons/images/icon_ereaser.png</iconset>
   </property>
   <property name="text">
    <string>Erease</string>
   </property>
  </action>
  <action name="actionWindowTimer">
   <property name="text">
    <string>Timer</string>
   </property>
  </action>
  <action name="actionWindowsMenu">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_windows.png</normaloff>:/icons/images/icon_windows.png</iconset>
   </property>
   <property name="text">
    <string>Window Menu</string>
   </property>
  </action>
  <action name="actionOpenPDF">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_pdf.png</normaloff>:/icons/images/icon_pdf.png</iconset>
   </property>
   <property name="text">
    <string>Open PDF</string>
   </property>
  </action>
  <action name="actionPdfNextPage">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_next.png</normaloff>:/icons/images/icon_next.png</iconset>
   </property>
   <property name="text">
    <string>Pdf Next Page</string>
   </property>
  </action>
  <action name="actionPdfPrevPage">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/icon_prev.png</normaloff>:/icons/images/icon_prev.png</iconset>
   </property>
   <property name="text">
    <string>Pdf Prev Page</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
